gooey/gui/presenter.py

```diff
@@ -65,9 +65,23 @@ def initialize_view(self):
       self.on_start()
     self.syncronize_from_model()
 
+  def create_model(self):
+    for group in self.model.groups():
+      if self.model.args(group):
+        self.view.create_section(group)
+        self.view.section(group).clear()
+        self.view.section(group).populate(self.model.args(group), self.model.num_cols_dict.get(group, self.model.num_default_cols))

```

Comment:
Fair enough. (I use an editor that does smart wrap, so tend not to notice these things.)